CRM/Campaign/Form/Task/Reserve.php

```diff
@@ -70,7 +70,7 @@ public function preProcess() {
     CRM_Campaign_BAO_Survey::retrieve($params, $this->_surveyDetails);
 
     //get the survey activities.
-    $activityStatus = CRM_Core_PseudoConstant::activityStatus('name');
```

Comment:
@demeritcowboy these swaps are safe aren't they? This PR is stale & I think turned out to be way trickier than initial appearance but I'm thinking maybe I can salvage some of the bits that are straight foward

Reply:
Some of them might be.